annotation/tests/test_distribution.py

```diff
@@ -1277,3 +1283,191 @@ def test_get_page_number_combinations(
     expected: List[int],
 ) -> None:
     assert get_page_number_combinations(file_pages, user_pages) == expected
+
+
+# ADDITIONAL TESTS
+
+USERS = [
+    {
+        "user_id": "405ef0e2-b53e-4c18-bf08-c0871615d99d",
+        "default_load": 100,
+        "overall_load": 0,
+    },
+    {
+        "user_id": "ba3e0ccf-8661-4ed3-892b-7c291160f631",
+        "default_load": 96,
+        "overall_load": 0,
+    },
```

Comment:
Do you need so much users for your tests? Usually you would have something normal and edge cases, apply this principle here as well. And make it a fixture, please. Same applies to ANNOTATION_TASKS_CHOOSE_VALIDATORS